querybook/server/lib/ai_assistant/base_ai_assistant.py

```diff
@@ -314,10 +331,29 @@ def query_auto_fix(
         query = query_execution.query
         language = query_execution.engine.language
 
+        self._query_auto_fix(
+            socket=socket,
+            metastore_id=query_execution.engine.metastore_id,
+            language=language,
+            query=query,
+            error=self._get_query_execution_error(query_execution),
+            session=session,
+        )
+
+    @with_session
+    def _query_auto_fix(
+        self,
+        socket,
```

Comment:
position of this socket is kind of all over the space, consider moving it to be before session